src/community/geopkg/src/test/java/org/geoserver/geopkg/wps/GeoHashCollectionTest.java

```diff
@@ -9,13 +9,28 @@
 import org.geotools.data.DataTestCase;
```

Comment:
New file without copyright header. Checkstyle does not run on community modules, but it would still be nice to have headers in place.